core/src/main/java/io/fabric8/maven/core/service/openshift/OpenshiftBuildService.java

```diff
@@ -515,6 +483,24 @@ private void logBuildBuildFailedDetails(OpenShiftClient client, String buildName
         }
     }
 
+    private void logBuildFailure(OpenShiftClient client, String buildName) throws Fabric8ServiceException {
+        try {
+            List<Build> builds = client.builds().inNamespace(client.getNamespace()).list().getItems();
```

Comment:
Can't we query build by `name`?

Reply:
`Build` object has a slightly different name than buildconfig, for example in the above shared logs, it comes up as vertx-http-booster-test-updated-container-s2i-8